website/views/bounty.py

```
@@ -0,0 +1,176 @@
+from django.http import JsonResponse
+from django.views.decorators.csrf import csrf_exempt
+from django.views.decorators.http import require_POST
+
+from website.models import Contributor, GitHubIssue, Repo, UserProfile
+
+logger = logging.getLogger(__name__)
+
+
+@csrf_exempt
+@require_POST
+def bounty_payout(request):
```

Pylint (3.3.7)

[refactor] Too many return statements (11/6)

(R0911)